test/tests.py

```diff
@@ -723,6 +738,20 @@ def test(self):
         if status != 0: # No use compiling if dmlc failed
             return
 
+        # Check profile data
```

Comment:
Better to keep this code in a separate CTestCase subclass (e.g., you could override test() or run_dmlc())

Reply:
Ok. I introduced DMLCProfileTestCase, a subclass of CTestCase.